src/lib.rs

```diff
@@ -16,6 +16,10 @@ pub struct ValidatedArgs {
     #[cfg(target_os = "linux")]
     pub symlink_path: Option<String>,
     pub nodelay: bool,
+    #[cfg(feature = "simulated_output")]
+    pub sim_paths: Vec<PathBuf>,
```

Comment:
It doesn't look to me that these are useful on creation when passed into `struct Kanata` so it would be better to isolate it into `filesim.rs` and not have the change here.

Reply:
tried that, but then that bleeds into the signature of `fn cli_init()`, which would return `(ValidatedArgs,cfg_paths,sim_paths)` or just `ValidatedArgs` depending on the feature, so moved it to the struct fields
but can try this↓ see commit